### PR TITLE
[DevNet] tweaks for remote environments

### PR DESCRIPTION
**Pull request by okdas**
## Description

Some tweaks necessary to run DevNet.

## Issue

https://github.com/pokt-network/pocket/issues/307

## Type of change

Please mark the relevant option(s):

- [x] Documentation
- [x] Tooling

## List of changes

- Add `cluster-manager` to the container image
- Add changes to the documentation

## Testing

- [x] verify validators work on DevNet
- [ ] `make develop_test`
- [ ] [LocalNet](https://github.com/pokt-network/pocket/blob/main/docs/development/README.md) w/ all of the steps outlined in the `README`

## Required Checklist

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added, or updated, [`godoc` format comments](https://go.dev/blog/godoc) on touched members (see: [tip.golang.org/doc/comment](https://tip.golang.org/doc/comment))
- [x] I have tested my changes using the available tooling
- [x] I have updated the corresponding CHANGELOG

### If Applicable Checklist

- [ ] I have updated the corresponding README(s); local and/or global
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added, or updated, [mermaid.js](https://mermaid-js.github.io) diagrams in the corresponding README(s)
- [ ] I have added, or updated, documentation and [mermaid.js](https://mermaid-js.github.io) diagrams in `shared/docs/*` if I updated `shared/*`README(s)

### CODE REVIEW

File: docs/development/README.md [hunk 1]
```diff
@@ -243,6 +244,10 @@ make client_start && make client_connect
 ✔ TriggerNextView # Commit the transaction
 ```
 
+### Creating a container image from developer's branches
```
Comment (Olshansk) [inline docs/development/README.md:247]:
```suggestion
### Build a container image from development's branch
```

File: docs/development/README.md [hunk 2]
```diff
@@ -243,6 +244,10 @@ make client_start && make client_connect
...
+
```
Comment (Olshansk) [inline docs/development/README.md:248]:
```suggestion
_tl;dr Attach the `push-image` label to the PR to build and push an image from your branch_ 

```

File: docs/releases/README.md [hunk 3]
```diff
@@ -26,20 +26,19 @@ For example, `X.Y.Z[-<pre_release_tag][+branch_name][+short_hash][+dirty]` is th
...
-Our images are hosted on Github's Container Registry (GHCR) and are available at `ghcr.io/poktnetwork/pocket-v1`. You can find the list of latest images [here](https://github.com/pokt-network/pocket/pkgs/container/pocket-v1).
+Our images are hosted on GitHub’s Container Registry (GHCR) and are available at `ghcr.io/poktnetwork/pocket-v1`. You can find the list of latest images [here](https://github.com/pokt-network/pocket/pkgs/container/pocket-v1).
 
 ### Tags
 
-Code built from the default branch (i.e. `main`) is tagged as `latest`.
+Code built from the default branch (i.e., `main`) is tagged as `latest`.
 
-Code built from commits in Pull Requests, is tagged as `pr-<number>`, as well as `sha-<7 digit sha>`.
+Code built from commits in Pull Requests, is tagged as `pr-<number>`, as well as `sha-<7 digit sha>`. If you need to build an image from a Pull Request, attach a `push-image` label to the pull request to force CI to upload a container image from a non-main branch.
```
Comment (Olshansk) [inline docs/releases/README.md:35]:
```suggestion
Code built from commits in Pull Requests, is tagged as `pr-<number>`, as well as `sha-<7 digit sha>`. You can optionally build an image from a Pull Request by attaching a `push-image` label to the pull request to force CI to upload a container image from a non-main branch.
```

File: build/Dockerfile.debian.dev [hunk 4]
```diff
@@ -53,6 +53,9 @@ RUN make protogen_local && \
 
 RUN go get -d -v ./app/pocket
 RUN go build -o /usr/local/bin/pocket ./app/pocket
-RUN go build -o /usr/local/bin/client ./app/client
+RUN go build -tags=debug -o /usr/local/bin/client ./app/client
+
+# TODO(@okdas): remove once cluster-manager functionality migrated to pocket-operator
```
Comment (Olshansk) [inline build/Dockerfile.debian.dev:58]: Is this something we even plan on doing? If not, let's remove it.

File: build/localnet/README.md [hunk 5]
```diff
@@ -24,28 +25,49 @@ This guide shows how to deploy a LocalNet using [pocket-operator](https://github
   - [Full Cleanup](#full-cleanup)
 - [Code Structure](#code-structure)
 
+
+## TLDR
```
Comment (Olshansk) [inline build/localnet/README.md:29]: nice

**Review by Olshansk — DISMISSED** [review]

File: docs/releases/README.md [hunk 6]
```diff
@@ -26,20 +26,19 @@ For example, `X.Y.Z[-<pre_release_tag][+branch_name][+short_hash][+dirty]` is th
...
-Our images are hosted on Github's Container Registry (GHCR) and are available at `ghcr.io/poktnetwork/pocket-v1`. You can find the list of latest images [here](https://github.com/pokt-network/pocket/pkgs/container/pocket-v1).
+Our images are hosted on GitHub’s Container Registry (GHCR) and are available at `ghcr.io/poktnetwork/pocket-v1`. You can find the list of latest images [here](https://github.com/pokt-network/pocket/pkgs/container/pocket-v1).
 
 ### Tags
 
-Code built from the default branch (i.e. `main`) is tagged as `latest`.
+Code built from the default branch (i.e., `main`) is tagged as `latest`.
 
-Code built from commits in Pull Requests, is tagged as `pr-<number>`, as well as `sha-<7 digit sha>`.
+Code built from commits in Pull Requests, is tagged as `pr-<number>`, as well as `sha-<7 digit sha>`. You can optionally build an image from a Pull Request by attaching a `push-image` label to the pull request to force CI to upload a container image from a non-main branch.
```
Comment (dylanlott) [inline docs/releases/README.md:35]: Should protocol-infra docs reference this convention?

**Review by dylanlott — APPROVED** [review]
LGTM! 🔎

File: docs/releases/README.md (hunk 6 above)
Comment (okdas) [reply docs/releases/README.md:35]: @dylanlott Haha, they do: https://github.com/pokt-network/protocol-infra#how-to-run-a-pocket-version-from-the-developers-branch

**Pull request merged**